RichEditorView/Assets/editor/rich_editor.js

```diff
@@ -372,3 +373,67 @@ RE.getSelectedHref = function() {
 
     return href ? href : null;
 };
+
+/* Make sure all text nodes are wrapped in divs! */
+
+RE.wrapTextNodes = function() {
```

Comment:
Is there a reason all the nodes have to be wrapped in divs? I think I told you that it was a specification for the project that I made this on, I don't really remember. I'm not saying you need to take it out, I'm just curious.